src/contracts.js

```diff
@@ -277,14 +277,12 @@
         (x) => x + "th"
     }
 
-    function pluralize {
-        (0, str) => str + "s",
-        (1, str) => str,
-        (n, str) => str + "s"
+    function pluralize(a0, str) {
+        return a0 === 1 ? str : (str + "s");
     }
 
     function toContract(f) {
-        return check(f, if f.name then f.name else "custom contract");
+        return check(f, function () { return f.name ? f.name : "custom contract"; });
```

Comment:
should be invoking this function. Or perhaps just fix the `if` macro? It's in helper-macros.js.